src/components/TextField/components/Resizer/tests/Resizer.test.tsx

```diff
@@ -0,0 +1,112 @@
+import * as React from 'react';
+import {mountWithAppProvider, findByTestID, trigger} from 'test-utilities';
+import {noop} from '@shopify/javascript-utilities/other';
+import Resizer from '../Resizer';
+import EventListener from '../../../../EventListener';
+
+describe('<Resizer />', () => {
+  const mockProps = {
+    onHeightChange: noop,
+    contents: 'Contents',
+  };
+
+  describe('contents', () => {
+    it('renders contents', () => {
+      const contents = 'Contents';
```

Comment:
I feel like in this test it can probably just be text contents for clarity sake.